README.md

```diff
@@ -46,9 +46,9 @@ Every included resource **MUST** reference resources included as primary data ei
 
 All resources, which are referenced by another resource, **MUST** either have [client-generated IDs](https://jsonapi.org/format/#crud-creating-client-ids) or a `lid` to locally [identify the resource](https://jsonapi.org/format/#document-resource-object-identification) within the document.
 
-### Example document
+### Example request
```

Comment:
I would prefer keeping "document" in the title. It's not showing an entire request but only the document send as request payload. Maybe "Request document example" and "Response document example"?